### DISCUSSION TITLE
Insert own pic?

### DISCUSSION THREAD

**Discussion by Stenfelt** [Acly/krita-ai-diffusion#931] [General]
Hello, 
I'm new to Krita AI Diffusion; I understand how to remove and replace objects from a marked area but I don't find how to insert an own picture... So if I mark a square I can then write "pot" and a pot will show up. But I would like to add my own specific pot picture. Please help. :)

**Comment by XylitolJ** [comment]
You can insert image into Krita as normal (drag and drop -> insert as a new layer), remove background of image if needed, then select area of image -> down strength to Refine that area (combine controlnet if needed or maybe not, just adjust value of strength)

link for plugin of auto select (also made by Acly):
https://github.com/Acly/krita-ai-tools

![Screenshot 2024-07-11 142333](https://github.com/Acly/krita-ai-diffusion/assets/16879990/ca15c174-bb0f-45a6-8622-730853fe203f)
